pkg/linkValidator/linkValidator.go

```diff
@@ -0,0 +1,82 @@
+package linkValidator
```

Comment:
The canonical package name format in Go is snake case, so this should be `package link_validator` and then update imports with the new name. The folder structure should also be `pkg/link_validator`